oak_functions/loader/src/main.rs

```diff
@@ -238,12 +238,32 @@ async fn async_main(opt: Opt, config: Config, logger: Logger) -> anyhow::Result<
 }
 
 async fn load_lookup_data(config: &Config, logger: Logger) -> anyhow::Result<Arc<LookupData>> {
+    let lookup_data_source = if config.lookup_data_url.is_empty() {
```

Comment:
Can `lookup_data_download_period` be removed from the examples that don't need it. I think previously we had to include them even for the examples that did not use lookup data, like mobilenet and perhaps also metrics.